Sources/Testing/Attachments/AttachableImageFormat.swift

```diff
@@ -0,0 +1,93 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024–2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+/// A type describing image formats supported by the system that can be used
+/// when attaching an image to a test.
+///
+/// When you attach an image to a test, you can pass an instance of this type to
```

Comment:
It might be helpful to elaborate a little bit on what "image" means here - specifically, that it's some structured representation of an image that's something other than a generic `Data` / bag of bytes, and that `AttachableImageFormat` is involved in configuring the process of transforming it into the `Data`-style representation.